rest-service/manager_rest/resolver_with_catalog_support.py

```diff
@@ -100,16 +101,57 @@ def _resolve_plugin_yaml_url(self, import_url):
 
     @staticmethod
     def _find_plugin(name, filters):
+        def _get_specifier_set(package_versions):
+            # Flat out the versions, in case one of them contains a few
+            # operators/specific versions
+            _versions = (v for vs in package_versions for v in vs.split(','))
+            specs = SpecifierSet()
+            for spec in _versions:
+                if not spec:
+                    raise InvalidSpecifier()
+                try:
+                    specs &= SpecifierSet(spec)
+                except InvalidSpecifier:
+                    # If the code below doesn't raise any exception then it's
+                    # the case where a version has been provided with no
+                    # operator to prefix it.
+                    specs &= SpecifierSet('==={}'.format(spec))
```

Comment:
should this be `===` (three equals) or `==` (two equals)?

Reply:
In the old version it would pick the plugin with exactly what's inside `spec`. Therefore `===`.